packages/terra-form-select/src/shared/_MenuUtil.js

```diff
@@ -198,6 +198,18 @@ class MenuUtil {
     return index === -1 ? null : options[Math.min(index + 1, options.length - 1)].props.value;
   }
 
+  /**
+   * Finds the index of option following the active option.
+   * @param {ReactNode} object - The node being flattened.
+   * @param {string} value - The value of the active option.
+   * @return {integer} - The value of the index of option.
+   */
+  static checkIndex(object, value) {
```

Comment:
This isn't an exposed API so the naming isn't a big deal, but ordinarily I'd expect this to be something like `findIndex` or `getIndex`.

Reply:
Done 1369bb9